src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -201,11 +202,9 @@ describe('sidebar.components.sidebar-content', function() {
     });
 
     it('clears the directLinkedGroupFetchFailed state', () => {
-      ctrl.directLinkedGroupFetchFailed = true;
```

Comment:
The logic to set the flag has been removed, but it hasn't been replaced with anything else. Did you mean to put the `store.setDirectLinkedGroupFetchFailed()` call in this test instead of the one above?

Reply:
agh yes!